### ISSUE TITLE
Ultra fast lane detection model to unit and munit

### ISSUE THREAD

**Issue by anita-hu** [anita-hu/sim2real-lane-detection#4]
loss functions (non cyclic and cyclic) added to the generator update in trainers
small modifications to train.py based on dataloader
When task 2 is done need to add to DRIT also
